File: src/gestorAplicacion/administracion/CategoriaHabitacion.java
/* Autores: Diego Andres Gracia Granados, Daniel Giraldo Vanegas, Elian David Velandia Riveros, Juan Camilo Gutierrez Martinez
y Santiago Arboleda Acevedo */
package gestorAplicacion.administracion;

public enum CategoriaHabitacion
{
    CAMILLA(50000),
    INDIVIDUAL(150000),
    DOBLE(320000),
    OBSERVACION(500000),
    UCI(1300000),
    UCC(1500000);
    private final int valor;

    private CategoriaHabitacion(int valor)
    {
        this.valor=valor;
    }

    public int getValor()
    {
        return valor;
    }
}

File: Python/src/gestor_aplicacion/administracion/categoria_habitacion.py
# Autores: Diego Andres Gracia Granados, Daniel Giraldo Vanegas, Elian David Velandia Riveros, Juan Camilo Gutierrez Martinez
# y Santiago Arboleda Acevedo
from enum import Enum
#enum que contiene las categorias de las habitaciones y sus precios
class Categoria_habitacion(Enum):
    CAMILLA = 50000
    INDIVIDUAL = 150000
    DOBLE = 320000
    OBSERVACION = 500000
    UCI = 1300000
    UCC = 1500000

    def __init__(self, valor):
        self.valor = valor

    def get_valor(self):
        return self.valor